config/locales/de.yml

```diff
@@ -395,7 +395,7 @@ de:
         experience: Erfahrung
         family: Familie
         german_course: Deutsch-Kurs
```

Comment:
I guess this and all other columns that were removed should also get removed (like zurich) from (both) translation files. I think those are: other_offer, dancing, health, cooking, excursions, women, sport, creative, music, culture, training, german_course, teenagers, children.

Reply:
done